server/accounts.go

```diff
@@ -4074,31 +4078,42 @@ func (dr *DirAccResolver) Start(s *Server) error {
 		if len(tk) != accLookupReqTokens {
 			return
 		}
-		if theJWT, err := dr.DirJWTStore.LoadAcc(tk[accReqAccIndex]); err != nil {
-			s.Errorf("Merging resulted in error: %v", err)
+		accName := tk[accReqAccIndex]
+		if theJWT, err := dr.DirJWTStore.LoadAcc(accName); err != nil {
+			if errors.Is(err, fs.ErrNotExist) {
+				s.Debugf("RESOLVER - Could not find account %q", accName)
+				go func() {
+					// Reply with empty response to signal absence of JWT to others,
+					// but not too fast to mitigate beating actual responses with content.
```

Comment:
This feels brittle and time sensitive based on rtt between entities in the system.

I think if the resolver is in partial or cache mode we stay silent, but if in full mode we all should have the same accounts so we should error (return nil) right away.

Reply:
Updated to be 1/10th of the configured fetch timeout, with the default of 1.9s that we have set [here](https://github.com/nats-io/nats-server/blob/main/server/const.go#L210) this means that the resolvers in full mode without all state would back off for 190ms before sending signal that they do not have it.